specification/monitor/resource-manager/readme.md

```diff
@@ -28,7 +28,30 @@ These are the global settings for the MonitorClient API.
 title: MonitorClient
 description: Monitor Management Client
 openapi-type: arm
-tag: package-2017-09
+tag: package-2018-02
+```
+### Tag: package-2018-02
+

```

Comment:
Since there are both stable and preview swaggers, should this package be a preview?

Reply:
What this package mean, is it the SDK? if so, it should remain preview. 
Please do not merge after your signoff, since the GA date for the api is not til Feb 20th. That's when we would like the swagger spec to be published.

Reply:
Yes, I have seen the publish date :) Thanks for adding this info from the very beginning, actually :D
The tag will become an SDK tag, yes, but if it includes preview APIs (I assume preview APIs might change anytime), the best would be to mark it as a preview package/tag.

---
In reply to: [166483608](https://github.com/Azure/azure-rest-api-specs/pull/2435#discussion_r166483608) [](ancestors = 166483608)

Reply:
resolved